key-manager/src/main/scala/org/bitcoins/keymanager/InitializeKeyManagerError.scala

```diff
@@ -1,21 +1,16 @@
-package org.bitcoins.wallet.api
+package org.bitcoins.keymanager
 
-sealed trait InitializeWalletResult
+sealed trait InitializeKeyManagerError extends Error
```

Comment:
nit: `sealed abstract class InitializeKeyManagerError(msg: String) extends Error(msg)` or something like that should let you do things like `extends InitializeKeyManagerError("Bad Entropy")` below ... I think